_src/bundle.css

```diff
@@ -177,7 +177,7 @@ padding-left: 10px;}
 .dropdown-content {
     display: none;
     position: absolute;
-    background-color: rgb(255, 238, 179, 0.8);
+    background-color: rgb(255, 255, 255, 0.95);
```

Comment:
@SavannahRicks Here's where we altered the opacity and changed the background color.